Pipfile

```diff
@@ -9,6 +9,7 @@ verify_ssl = true
 openshift = "==0.11.2"
 jupyterhub-singleuser-profiles = "==0.4.2"
 oauthenticator = {ref = "d98936da37569b7bb400c6854b07041dce3e7f87",git = "https://github.com/opendatahub-io/oauthenticator.git", editable = true}
+jupyterhub-traefik-proxy = {ref = "b468da04bf0057e9baeb458de6c3f9f677cfb79e", git = "https://github.com/opendatahub-io/traefik-proxy.git", editable = true}
```

Comment:
Shouldn't this refer to rhds traefik proxy?

Reply:
Depends, so its exactly the same code as upstream, in which case shouldn't we be using upstream libraries? 🤔

Reply:
I'm leaning towards it anyways. @crobby  / @tmckayus  whaddya think?

Reply:
I was just thinking about this. We should probably fork it, we have with everything else.

Reply:
One thing to note here, though -- CPaaS doesn't use Pipfile. It uses the requirements*.txt

So this change will have no effect for a CPaaS build, only an upstream build of this repo. To pick it up in CPaaS, we need the requirements.txt altered.

Reply:
ack yeah I did not know that
Ill update that file!

Reply:
Updated